frontend/src/api.ts

```diff
@@ -0,0 +1,29 @@
+import { FormData } from './components/Form/Form'
+
+export const handleSubmitForm = async (e: React.FormEvent<HTMLFormElement>, formData: FormData) => {
+  e.preventDefault()
+
+  if (formData.name === '' || formData.age === '' || formData.favoriteColor === '') {
+    return
+  }
+
+  try {
+    const response = await fetch(`${import.meta.env.VITE_API}//survey/submit`, {
```

Comment:
this double "//" is wrong anyway we should use more save way to build path

Reply:
This  file has double logic (just api client and validation we should split this to avoid possible code issues)